src/services/WorkflowHandler.ts

```diff
@@ -172,11 +185,11 @@ export function getSolveNode(solverModel: Runnable, outputHandler: Function) {
       const chain = chatPromptTemplate.pipe(solverModel);
 
       const responseResult = await chain.invoke({task: state.task, plan: plan, results: state.results});
-
+      
```

Comment:
Delete space

Reply:
Was this added by prettier, if that's the case I'm wrong and we can keep it as is

Reply:
Yeah, that's prettier